test/performance/odp_sched_perf.c

```diff
@@ -58,6 +61,7 @@ typedef struct test_options_t {
 	uint32_t tot_queue;
 	uint32_t tot_event;
 	int      touch_data;
+	int      stress;
```

Comment:
Could be `uint32_t` as  `strtoul()` returns `unsigned long`.

Reply:
Fixed in v3